target/linux/rockchip/patches-5.4/104-rockchip-use-USB-host-by-default-on-rk3399-rock-pi-4.patch

```diff
@@ -0,0 +1,33 @@
+From 75b1c522cc33fa9b79077802a49803c874f647b1 Mon Sep 17 00:00:00 2001
+From: Marty Jones <mj8263788@gmail.com>
```

Comment:
> I backported this patch from upstream to fix connections errors

Then please keep it as it is, now you've claimed yourself as the author of the patch which is not correct.

Reply:
I've corrected the mistake.